release/models/network-instance/openconfig-evpn.yang

```diff
@@ -1061,5 +1068,165 @@ module openconfig-evpn {
+
+        leaf peer-address {
+        type leafref{
+          path '../state/peer-address';
+        }
+        description "overlay endpoint peer ip address";
+        }
+        container state {
+         config false;
+         description
+           "State parameters relating to the overlay tunnel endpoints
+           of the network instance";
+           uses evpn-endpoint-peer-state;
+        }
+      }
+    }
+
+    container endpoint-vnis {
+      description "Overlay endpoint peers oper data";
+      config false;
+
+      list endpoint-vni {
+        key "vni";
+        description "Overlay endpoint evis oper data";
+
+        leaf vni {
+          type leafref{
+            path '../state/vni';
+            }
+            description "VNI Identifier";
+        }
+        container state {
+          config false;
+           description
+             "State parameters relating to the overlay tunnel endpoints
+             of the network instance";
+             uses evpn-endpoint-vni-state;
+        }
+      }
+    }
+  }
+
+  grouping evpn-endpoint-peer-state {
+    description
+    "Grouping for VXLAN EVPN Overlay endpoint oper data";
+    leaf peer-address {
+      type oc-inet:ip-address;
+      description "VXLAN EVPN Overlay endpoint peer ip address";
+    }
+    leaf peer-state {
+      type enumeration {
+        enum UP {
+          description "VXLAN EVPN Overlay peer state is UP";
+        }
+        enum DOWN {
+          description "VXLAN EVPN Overlay peer state is DOWN";
+        }
+      }
+      description "VXLAN EVPN Overlay peer state";
+    }
+    leaf uptime {
+      type yang:timestamp;
+      description "VXLAN EVPN Overlay Peer Uptime";
+    }
+    leaf control-plane-vnis {
+      type string;
+      description "VNIs learned over the control-plane";
+    }
+    leaf router-mac {
+      type oc-yang:mac-address;
+      description "Peer Router mac-address";
+    }
   }
+
+  grouping evpn-endpoint-vni-state {
+    description "Grouping for VXLAN EVPN Overlay endpoint to vni mapping data";
+    leaf vni {
+      type oc-evpn-types:evi-id;
+      description "VNI Identifier";
+    }
+    leaf multidestination-traffic {
+      type union {
+        type oc-evpn-types:multicast-address;
+        type enumeration {
+          enum STATIC_INGRESS_REPLICATION {
+            description
+              "Static ingress replication mode.";
+          }
+          enum BGP_INGRESS_REPLICATION {
+            description
+              "BGP EVPN ingress replication mode. It includes the ability to
+              signal a P2MP LSP for the EVPN Inclusive Provider Tunnel
+              for BUM traffic";
+          }
+        }
+      }
+      description
+        "The data plane for overlays needs to handle the transport of
+         multidestination traffic. Multidestination traffic is typically
+         referred to as “BUM,” which stands for broadcast, unknown unicast,
```

Comment:
Some toolchains do not like non-ASCII characters here -- can we fix the quotes here?

Reply:
Fixed